bugsnag-android-core/src/main/java/com/bugsnag/android/StrictModeHandler.java

```diff
@@ -87,12 +89,7 @@ String getViolationDescription(String exceptionMessage) {
      * @return the root cause of the throwable
      */
     private Throwable getRootCause(Throwable throwable) {
-        Throwable cause = throwable.getCause();
-
-        if (cause == null) {
-            return throwable;
-        } else {
-            return getRootCause(cause);
-        }
+        List<Throwable> causes = ThrowableExtensionsKt.safeUnrollCauses(throwable);
```

Comment:
I think your suggestion of using `@file:JvmName("ThrowableUtils")` to rename the file would work well here.